playbooks/roles/airship-configure-caasp/tasks/main.yml

```diff
@@ -29,5 +32,8 @@
 
 - name: Create privileged ClusterRoleBinding in CaaSP
   command: "kubectl apply -f /tmp/privileged-cluster-role-binding.yml --overwrite=true"
+  register: create_privileged_cluster_role_binding_result
+  changed_when: "create_privileged_cluster_role_binding_result.stdout|search('created')"
   tags:
     - run
+    - skip_ansible_lint
```

Comment:
same question.

Reply:
thx @evrardjp I should have removed 'skip_ansible_lint' tag, I am not getting any error when I remove it. Will handle this as a new PR since this PR was already merged